src/utils/currency.ts

```diff
@@ -1,3 +1,33 @@
 import { currencyKeyToIconMap, CurrencyKey } from 'constants/currency';
+import { getEthereumNetwork } from 'utils/networkUtils';
+
+const { SynthetixJs } = require('synthetix-js');
+
+interface ICurrencyDetails {
+	tokenAddress: string;
+	tokenSymbol: string;
+	tokenDecimals: number;
+	tokenImage: string;
+}
 
 export const getCurrencyKeyIcon = (currencyKey: CurrencyKey) => currencyKeyToIconMap[currencyKey];
+
+export const getCurrencyKeyURLPath = (currencyKey: CurrencyKey) =>
+	`https:///www.synthetix.io/assets/synths/svg/${currencyKey}.svg`;
+
+export const getCurrencyDetails = async (currencyKey: CurrencyKey): Promise<ICurrencyDetails> => {
+	// Uses default setup - can't find anything in synthetix docs to set this to testnet
```

Comment:
I think we can drop this function since we can grab all the details from `snxJSConnector`. But keep `getCurrencyKeyURLPath` because we need it for the image.